traffic_portal/app/src/common/modules/form/user/form.user.tpl.html

```diff
@@ -82,6 +82,14 @@
                     <small ng-show="user.tenantId"><a href="/#!/tenants/{{user.tenantId}}" target="_blank">View Details&nbsp;&nbsp;<i class="fa fs-xs fa-external-link"></i></a></small>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(userForm.uCDN), 'has-feedback': hasError(userForm.uCDN)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Upstream CDN</label>
```

Comment:
Label should have a `for` attribute with a value set to the ID of the labeled control